pkg/generator/helper/helper_test.go

```diff
@@ -25,6 +25,13 @@ func configOpenFileMock(name string) (io.ReadCloser, error) {
 spec:
   outputs:
     resources:
+      - name: reports
```

Comment:
@nevskrem what do you think? Does this work for Security steps?

Reply:
As long as we are able to specify path and file pattern it should be able to cover the security related ones.